In [1]:
import numpy as np
import pandas as pd
from sklearn.preprocessing import LabelEncoder
from sklearn.preprocessing import StandardScaler
import pickle
from sklearn.model_selection import train_test_split

In [2]:
from Deployment_Refactored_Version import MaterialClassifier

In [3]:
classifier = MaterialClassifier()

In [4]:
# Provide input data
su_input = float(input("Ultimate Tensile Strength (Su) in MPa:"))
E_input = float(input("Elastic Modulus (E) in MPa:"))
g_input = float(input("Shear Modulus (G) in MPa:"))
ro_input = float(input("Density (Ro) in Kg/m3:"))

Ultimate Tensile Strength (Su) in MPa:1000
Elastic Modulus (E) in MPa:119876
Shear Modulus (G) in MPa:67543
Density (Ro) in Kg/m3:8912


In [5]:
input_data = [su_input, E_input, g_input, ro_input]

# Use the classifier to make predictions
y = classifier.predict(input_data)

if y == 0:
    print("Future Prediction = [False]")
else:
    print('Future Prediction = [True]')

Future Prediction = [False]


/Users/viswanathanmuthu/anaconda3/envs/AIVE_Course/lib/python3.11/site-packages/sklearn/base.py:439: UserWarning: X does not have valid feature names, but StandardScaler was fitted with feature names
  warnings.warn(


Ultimate Tensile Strength (Su) in MPa:1000
Elastic Modulus (E) in MPa:119876
Shear Modulus (G) in MPa:67543
Density (Ro) in Kg/m3:8912